src/geometry/narrow_phase.rs

```diff
@@ -455,12 +455,34 @@ impl NarrowPhase {
                             }
                         }
                     }
-
+                    // Remove the new parent from the collision graphs
```

Comment:
```suggestion
                    // Remove the new parent from the contact and intersection graphs
```

Also, shouldn´t we remove that from graph_indices too?